packages/terra-open-source-scripts/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@cerner/terra-open-source-scripts",
+  "private": true,
+  "version": "0.0.0",
+  "description": "Terra package with various scripts for open source development",
+  "main": "index.js",
+  "bin": {
+    "terra": "./bin/terra"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-toolkit.git",
+    "directory": "packages/terra-open-source-scripts"
+  },
+  "keywords": [
+    "terra-open-source-scripts",
+    "Cerner",
+    "Terra"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-toolkit/issues"
+  },
+  "engines": {
+    "node": "^10.13.0"
+  },
+  "homepage": "https://github.com/cerner/terra-toolkit",
+  "scripts": {
+    "compile": "babel src --out-dir lib --copy-files --ignore src/terra-cli",
+    "lint": "eslint --ext .js,.jsx ."
+  },
+  "dependencies": {
+    "@cerner/terra-cli": "^0.0.0",
```

Comment:
Shouldn't this be a dev-dependency and peer-dependency?

I expected "@cerner/terra-cli" to be added directly by the project want to use it's commands

Reply:
I need it for the cli logging so it definitely needs to be some form of dependency.  I'm not sure peer needs to happen.  What are your concerns with making it a direct dependency?

Reply:
I think i agree with making it a peer dependency. I think of terra-open-source-scripts as a plugin to the terra-cli package. So the user 'opts' into terra-cli then adds the plugins to add scripts.

Reply:
Fair enough and I don't have a strong preference necessarily

Reply:
https://github.com/cerner/terra-toolkit/pull/460/commits/6c35f4191a2115b9652c22dc9f42105554d0b08c